service/src/test/java/com/codeforcommunity/JooqMock.java

```diff
@@ -10,25 +14,27 @@
 import java.util.Objects;
 import java.util.function.Supplier;
 import org.jooq.DSLContext;
+import org.jooq.Field;
 import org.jooq.Record;
 import org.jooq.Result;
 import org.jooq.SQLDialect;
 import org.jooq.Table;
 import org.jooq.generated.DefaultSchema;
 import org.jooq.impl.DSL;
-import org.jooq.tools.jdbc.MockConnection;
-import org.jooq.tools.jdbc.MockDataProvider;
-import org.jooq.tools.jdbc.MockExecuteContext;
-import org.jooq.tools.jdbc.MockResult;
+import org.jooq.tools.jdbc.*;
```

Comment:
No * imports (your IDE probably changed it automatically)

Reply:
Made the change!